cmd/traffic/cmd/manager/internal/mutator/watcher.go

```diff
@@ -621,12 +683,25 @@ func (c *configWatcher) affectedConfigs(ctx context.Context, svc *core.Service,
 			wls = append(wls, stss...)
 		}
 	}
+
+	c.RLock()
```

Comment:
@thallgren do you have an opinion on this change ? It was the main bottle neck but I'm not sure if it can have an impact to reduce the scope of this lock.

Reply:
Great :)